sdv/constraints/tabular.py

```diff
@@ -221,40 +223,50 @@ class GreaterThan(Constraint):
             by checking if the value provided is a column name.
     """
 
-    _diff_column = None
+    _column_list = False
+    _diff_columns = None
     _is_datetime = None
     _column_to_reconstruct = None
 
     def __init__(self, low, high, strict=False, handling_strategy='transform',
                  fit_columns_model=True, drop=None, high_is_scalar=None,
                  low_is_scalar=None):
+        self.constraint_columns = (low, high)
```

Comment:
I think this needs to be revisited, since `low` and `high` may also be lists.
We should also add all the possible validations such as the inability to pass both `high` and `low` as `lists` here, in the `__init__`.